packages/db/src/translate/executions.ts

```diff
@@ -0,0 +1,71 @@
+import Knex from 'knex';
+
+import { ExecutionRecord } from '@cumulus/types/api/executions';
+import { PostgresExecution } from '../types';
+// TODO move to common location
+const {
+  getParentExecutionCumulusId,
+  getAsyncOperationCumulusId,
+  getCollectionCumulusId,
+} = require('../../api/lambdas/sf-event-sqs-to-db-records/utils');
+
+/**
+ * Translate execution record from Dynamo to RDS.
+ *
+ * @param {AWS.DynamoDB.DocumentClient.AttributeMap} dynamoRecord
+ *   Source record from DynamoDB
+ * @param {AWS.DynamoDB.DocumentClient.AttributeMap} knex
+ *   Knex client
+ * @returns {PostgresExecutionRecord} - converted Execution
+ */
+export const translateApiExecutionToPostgresExecution = async (
+  dynamoRecord: ExecutionRecord,
+  knex: Knex
+): Promise<PostgresExecution> => {
+  // Map old record to new schema.
+  const translatedRecord: PostgresExecution = {
+    async_operation_cumulus_id: (
+      dynamoRecord.asyncOperationId ? await getAsyncOperationCumulusId(
+        dynamoRecord.asyncOperationId, knex
+      ) : undefined
+    ),
+    status: dynamoRecord.status,
+    tasks: JSON.stringify(dynamoRecord.tasks),
+    error: JSON.stringify(dynamoRecord.error),
+    arn: dynamoRecord.arn,
+    duration: dynamoRecord.duration,
+    original_payload: JSON.stringify(dynamoRecord.originalPayload),
+    final_payload: JSON.stringify(dynamoRecord.finalPayload),
+    workflow_name: dynamoRecord.name,
```

Comment:
I could be wrong but I think `url` is missing
```suggestion
    workflow_name: dynamoRecord.name,
    url: dynamoRecord.execution,
```

Reply:
Oh! Awesome catch! 

So I missed this because I didn't realize the old Dynamo `execution` field mapped to `url` in PG. I thought `url` was new to the new model (an assumption I didn't realize I had until right now).

Is there a better place to see this kind of thing? What would be nice is sample data or something close to it to verify all of my translations here. What I have been using to verify what the fields are:

- The types/api/executions.d.ts Dynamo model
- The pre-existing execution test data in `fakeExecutionFactoryV2` 
- The schema.svg for the new PG schema

(updating this mapping to your suggested change and updating tests)

Reply:
Yeah, I think that is pretty easy to miss! I know I've done that before. 😅 

What I have done when working on similar tickets is having the schema definition for the DynamoDB table and PG table open side by side and creating a mental mapping of them. I also go the extra step to write them down in my notebook to make sure that no field goes unaccounted for (but that's kind of extra and not necessary).

Then I go look for records in DynamoDB to check my understanding and see what actual test records look like. 

For the case of reviewing your PR, I didn't write anything down but just compared the schema.

Not sure if that's helpful but hopefully gives you insight onto how I go about it!